content/blog/introducing-kube2pulumi/index.md

```diff
@@ -0,0 +1,417 @@
+---
+title: "Introducing kube2pulumi: No More YAML"
+date: 2020-08-13
+meta_desc: Automatically convert Kubernetes YAML manifests to modern code in TypeScript, Python, C#, and Go.
+meta_image: yaml2code.png
+authors:
+    - levi-blackstone
+    - sashu-shankar
+tags:
+    - Kubernetes
+---
+
+Kubernetes users often joke about being "YAML engineers," and the pile of YAML seems to get deeper every day. Today,
+we're pleased to announce [kube2pulumi], a tool to automatically convert Kubernetes manifests into modern code! Instead
+of manipulating YAML directly, you can take advantage of the rich ecosystem of programming language tools to
+supercharge your productivity.
+
+<!--more-->
+
+Using programming languages to manage Kubernetes resources has many benefits: strong typing, linting, auto-completion,
+refactoring, testing, and the ability to use standard language features like variables, loops, conditions, and
+abstraction. Pulumi also supports [Helm], [Kustomize], and [YAML], but these tools don't take full advantage of the
+programming language. With `kube2pulumi`, you can realize these benefits without manual conversion work upfront.
+
+Check out an example of `kube2pulumi` in action, and then we'll take a closer look at the generated code.
+
+## Getting Started with kube2pulumi
+
+Let's start with a simple example: converting a Namespace. Here's the YAML manifest:
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: foo
+```
+
+`kube2pulumi` supports all of Pulumi's SDKs, so you can choose whatever language you like!
```

Comment:
Not sure "Pulumi's SDKs " is clear here - suggestion below - but may need more rewording.

```suggestion
`kube2pulumi` supports all of Pulumi's supported languages, so you can choose whatever language you like!
```